lib/tasks/demo.rake

```diff
@@ -36,7 +36,64 @@ desc 'Backup the database'
     threads.each(&:join)
 
   end
-  
+
+  desc 'Configure the users in the database for performance testing'
+  task setup_performance_test_users: :environment do
+    raise 'This task is only for use in a development environment' unless Rails.env == 'development' || ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK']
+
+    jurisdictions = Jurisdiction.all
+    if !(jurisdictions.size > 50)
+      puts ' Jurisdictions were not found! Make sure to run `PERFORMANCE=true bundle exec rake admin:import_or_update_jurisdictions`'
+      exit(1)
+    end
+
+    users = []
+
+    # Super User at the USA level
+    users << create_user("#{jurisdictions.first.unique_identifier}_super_user", Roles::SUPER_USER, jurisdictions.first)
```

Comment:
would it make more sense to use the jurisdiction `:path` instead of `:unique_identifier`? maybe something like:
```
jurisdictions.first[:path].gsub(', ', '_').gsub(' ', '').downcase
```
(same for all the other users)

Reply:
Since no one is really going to be viewing the information I wanted to save cpu cycles and just use an attribute that is already present instead of doing a lot of path manipulation

Reply:
is this mainly going to be used for the performance stuff in CI?

Reply:
Yes